viral/ensemble_reactivation.py

```
@@ -872,6 +874,11 @@ def compute_shuffled_strength(_):
             f"# significant components (Marcenko-Pastur), shuffled data: {num_shuffled_components if not use_cache else 'not computed'}"
         )
 
+    plot_reactivation_strength_change(
```

Ruff (0.13.3)

Undefined name `plot_reactivation_strength_change`

(F821)